code/controllers/subsystems/ticker.dm

```diff
@@ -275,14 +275,16 @@ Helpers
 /datum/controller/subsystem/ticker/proc/create_characters()
 	for(var/mob/new_player/player in global.player_list)
 		if(player && player.ready && player.mind)
```

Comment:
The typed loop above should make it impossible for `player` to be null here: removing the two player null checks would be a bit cleaner.